docs/nx-cloud/enterprise/dte/circle-ci-dte.md

```diff
@@ -0,0 +1,59 @@
+# Custom Distributed Task Execution on Circle CI
+
+Using [Nx Agents](/ci/features/distribute-task-execution) is the easiest way to distribute task execution, but it your organization may not be able to use hosted Nx Agents. With an [enterprise license](https://nx.app/enterprise), you can set up distributed task execution on your own CI provider using the recipe below.
+
+## Run Custom Agents on Circle CI
+
+Run agents directly on Circle CI with the workflow below:
+
+```yaml {% fileName=".circleci/config.yml" %}
+version: 2.1
+orbs:
+  nx: nrwl/nx@1.5.1
+jobs:
+  main:
+    docker:
+      - image: cimg/node:lts-browsers
+    steps:
+      - checkout
+      - run: npm ci
+      - nx/set-shas
+
+      # Tell Nx Cloud to use DTE and stop agents when the build tasks are done
+      - run: npx nx-cloud start-ci-run --stop-agents-after=build
```

Comment:
Same as above: We are missing the `NX_CLOUD_DISTRIBUTED_EXECUTION_AGENT_COUNT: 3` env variable

Reply:
And add `e2e-ci`